### PR TITLE
fix: llm palm score tests

### PR DESCRIPTION
**Pull request by ashleyxuu**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes internal # 336527025🦕

### CODE REVIEW

File: tests/system/load/test_llm.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def test_llm_palm_configure_fit(llm_fine_tune_df_default_index, llm_remote_text_
         model_name="text-bison", max_iterations=1
     )
 
-    df = llm_fine_tune_df_default_index.dropna()
+    df = llm_fine_tune_df_default_index.dropna().sample(n=100)
```
Comment (GarrettWu) [inline tests/system/load/test_llm.py:57]: Let's add retry for experimental features.

Comment (GarrettWu) [inline tests/system/load/test_llm.py:57]: curious how much run time reduction with sampling.

**Review by GarrettWu — APPROVED** [review]

File: tests/system/load/test_llm.py (hunk 1 above)
Comment (ashleyxuu) [reply tests/system/load/test_llm.py:58]: Done.

Comment (ashleyxuu) [reply tests/system/load/test_llm.py:58]: It didn't improve much. It still takes the same amount of time around 30-40 mins, and the durations has some variations.

**Pull request merged**